File: ClientApp/src/app/features/pickupPoints/classes/pickupPoint.ts
import { KeyValuePair } from 'src/app/shared/classes/keyValuePair'

export class PickupPoint extends KeyValuePair {

    description: string
    route: {
        id: number
        abbreviation: string
        description: string
    }
    exactPoint: string
    time: string
    coordinates: string
    isActive: boolean

}

File: BackEnd/Features/PickupPoints/Models/PickupPoint.cs
using System.Collections.Generic;
using BlueWaterCruises.Features.Reservations;
using BlueWaterCruises.Features.Routes;
using BlueWaterCruises.Infrastructure.Identity;

namespace BlueWaterCruises.Features.PickupPoints {

    public class PickupPoint {

        // PK
        public int Id { get; set; }
        // Fields
        public string Description { get; set; }
        public string ExactPoint { get; set; }
        public string Time { get; set; }
        public string Coordinates { get; set; }
        public bool IsActive { get; set; }
        // FKs
        public int RouteId { get; set; }
        public string UserId { get; set; }
        // Navigation
        public Route Route { get; set; }
        public UserExtended User { get; set; }
        public List<Reservation> Reservations { get; set; }

    }

}